scripts/Projector/flex_schedule_off.py

```diff
@@ -5,6 +5,6 @@
 
 def flex_schedule_off():
     ssh_connection = ssh.SSH(hostname, pi.username, pi.password)
-    ssh_connection.send_cmd("rm /tmp/flex")
+    ssh_connection.send_cmd("rm /home/pi/clock/public/flex-toggle")
     print("Done.")
```

Comment:
Can you provide better feedback, like "Flex schedule is now OFF"?  Also does send_cmd return anything, like  success or fail? I don't remember, if it does, you should check that first.

Reply:
resolved as per [42bd97a](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/50/commits/42bd97abdfbdc7122db7036a34528b17eded5852)